helm/alfresco-content-services/23.N_values.yaml

```diff
@@ -65,10 +65,10 @@ alfresco-control-center:
     tag: 9.3.0
 postgresql:
   image:
-    tag: 15.10.0
+    tag: 15.10
 postgresql-sync:
   image:
-    tag: 14.10.0
+    tag: 14.10
```

Comment:
I think it's safe to align with ACS version. the dsync CI even test again a latest tag of postgres